src/gameClasses/Player.js

```diff
@@ -643,7 +643,7 @@ var Player = IgeEntity.extend({
 			if (typeof (userId) !== 'undefined' && typeof (sessionId) !== 'undefined') {
 				if (ige.game.data.isDeveloper) {
 					// dont show dev menu by default
-					// if (ige.mobileControls && !ige.mobileControls.isMobile) {
+					// if (!ige.isMobile) {
```

Comment:
very thorough :D

Reply:
![unknown](https://user-images.githubusercontent.com/62058278/168298360-8bef2796-7761-4d7a-a7d4-0d2b95989561.png)
got this error with npm run debug command

Reply:
I think now that this function is immediately invoked, we will run into problems like this. 
`navigator.userAgent` is specifically a property on the window defined by the headers of the server request.
We might need an `isClient` check here and maybe even make sure to wait until navigator.userAgent is defined during page load.

Reply:
will address this

Reply:
moved assigning `isMobile` to init method, with condition that is is only on client